frontend/src/views/Members.vue

```diff
@@ -43,24 +43,22 @@ SPDX-License-Identifier: Apache-2.0
           Access to resources within your project can be configured by assigning roles.
         </p>
       </v-card-text>
-      <v-list two-line subheader v-else>
-        <template v-for="(user, index) in sortedAndFilteredUserList">
-          <v-divider v-if="index !== 0" inset :key="index"></v-divider>
+      <v-data-table
```

Comment:
not sortable, no pagination, no filter ==> <v-simple-table>?

Reply:
Filter is in place, sorting can be added if we want but I currently do not see a real use case for it... maybe for the roles but I don't know... I will add pagination and adjust the row spacing and alignment, as we discussed with @petersutter